Samples/Objective-C/nhubsample-refresh/nhubsample-refresh/AppDelegate.m

```diff
@@ -19,12 +19,37 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     NSString *connectionString = [configValues objectForKey:@"connectionString"];
     NSString *hubName = [configValues objectForKey:@"hubName"];
     
+    [MSNotificationHub setEnrichmentDelegate: self];
+    [MSNotificationHub setManagementDelegate: self];
+    [MSNotificationHub setLifecycleDelegate: self];
     [MSNotificationHub initWithConnectionString:connectionString hubName:hubName];
     [MSNotificationHub addTag:@"userAgent:com.example.nhubsample-refresh:1.0"];
     
     return YES;
 }
 
+- (void)notificationHub:(MSNotificationHub *)notificationHub willEnrichInstallation:(MSInstallation *)installation{
+    NSLog(@"willEnrichInstallation");
+}
+
+// Sample usage of MSInstallationManagementDelegate
```

Comment:
Can we make this easier to read by enclosing it in a multi-line comment?

Reply:
Fixed